xain/grpc/coordinator.py

```diff
@@ -20,55 +27,58 @@ class ParticipantContext:
 
     In the future we may store more information like in what state a participant is in e.g.
     IDLE, RUNNING, ...
+
+    Args:
+        participant_id (:obj:`str`): The id of the participant. Typically a
+            host:port or public key when using SSL.
     """
 
-    def __init__(self, participant_id):
+    def __init__(self, participant_id: str) -> None:
```

Comment:
Very minor comment, generally in this file: I think we don't need to mention the type again in the docstring if we have the `mypy` annotation (benefits in being in one place, etc.)

Reply:
I don't know exactly how sphinx deals with that. Specially if using intersphinx to link to external documentation

Reply:
It would be good if sphinx were smart enough with `mypy` annotations but I don't know. Anyway, this is a separate discussion and certainly no blocker here.